S23/sandeep-1-kumar/LowerCasing.java

```diff
@@ -0,0 +1,60 @@
+import java.util.Locale;
+
+public class LowerCasing {
+
+    public static void main(String[] args)
+    {
+
+    }
+    public String lowerCaseConversion(String languageType,String word)
+    {
+        String[] exceptedLanguages = {"zh" , "ja" , "th"};
+        Locale locale = Locale.forLanguageTag(languageType);
+        String language = locale.getLanguage();  
+        for(String element : exceptedLanguages)
+        {
+            if(element.equals(language))
+            {
+                return word;
+            }
+        }
+        String returningValue = word;
+        if(language.equals("tr") || language.equals("az"))
+        {
+            returningValue = word.replace('I' ,'\u0131');
+        }
+        else if(language.equals("el"))
+        {
+            if (word.endsWith("Σ"))
+            {
+                returningValue = word.substring(0, word.length() - 1) + "\u03c2"; 
+            }
+            else
+            {
+                returningValue = word.replace("Σ", "σ");
+            }
+        }
+        else if(language.equals("ga"))
```

Comment:
hope this test case is not handled : word : "nÓg"  language "ga-IE"

Reply:
here, I hope the check is implemented for lang "ga" but not for all lang belonging to type "ga", i.e "ga-IE"